Druid/DRInterpreterToCompiler.class.st

```diff
@@ -71,39 +97,67 @@ DRInterpreterToCompiler >> addPrimitives [
 DRInterpreterToCompiler >> build [
 	"Answer a <DRCompilationUnit>, useful for model manipulation before code dumping"
 
-	self compilationUnit compiler: self compilerCompiler.
+	self compilationUnit compiler: self primitiveCompiler.
 	self addPrimitives.
+	self compilationUnit compiler: self bytecodeCompiler.
```

Comment:
I don't like this statefull thing. We cannot change the compiler of the compilation unit like that. This is a recipee for disaster.
This indicates that the compilation unit should not know the compiler.